src/main/java/duke/DeleteCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+
+/**
+ * Deletes a task from  the task list
+ */
+public class DeleteCommand extends Command{
+
+    /** Index of the task in task list */
+    private final int INDEX;
```

Comment:
Saw that your commands used the variable name `INDEX`, perhaps it could be more specific? For example `INPUT_INDEX` or `TASK_INDEX` or something like that, which gives a bit more information about what this variable is about